coalib/bearlib/aspects/exceptions.py

```diff
@@ -0,0 +1,47 @@
+class aspectTypeError(TypeError):
```

Comment:
I think the name should be `AspectTypeError` as to follow python pep8 naming guideines.